portfolio/src/main/webapp/index.html

```diff
@@ -29,6 +29,12 @@ <h1>Gabi Newman's Portfolio</h1>
       <div id="fact-container"></div>
       <div id="comments">
         <h3>Comments</h3>
+        <form action="/data" method="POST">
+        <textarea name="comment-input">Your comment here</textarea>
```

Comment:
Is there a way to clear this when a user clicks in to edit it? Right now you have to manually delete it if you don't want it as part of your comment

Reply:
Suparna also pointed this out. I agree it's super annoying. I think it should be good now.